cuesubmit/cuesubmit/Submission.py

```diff
@@ -72,8 +73,14 @@ def buildBlenderCmd(layerData):
         renderCommand += ' -o {}'.format(outputPath)
     if outputFormat:
         renderCommand += ' -F {}'.format(outputFormat)
-    # The render frame must come after the scene and output
-    renderCommand += ' -f {frameToken}'.format(frameToken=Constants.FRAME_TOKEN)
+    if frameRange:
```

Comment:
It would be great if you could add unit tests to cover both the single-frame and start/end frame cases.

https://github.com/AcademySoftwareFoundation/OpenCue/blob/master/cuesubmit/tests/Submission_tests.py

Reply:
Added in https://github.com/AcademySoftwareFoundation/OpenCue/pull/1337/commits/2d4ba88b7a61c487a9e9454f3413e80eaae3deee

@bcipriano however, I'm having a bit of trouble with lines [#178](https://github.com/n-jay/OpenCue/blob/2d4ba88b7a61c487a9e9454f3413e80eaae3deee/cuesubmit/tests/Submission_tests.py#L178) and [#153](https://github.com/n-jay/OpenCue/blob/2d4ba88b7a61c487a9e9454f3413e80eaae3deee/cuesubmit/tests/Submission_tests.py#L178) of Submission_tests.py
The test fails when I set it to `blender` but passes when I set it to `shell`. This is despite the layerType being set as Blender. Any thoughts on this?

Reply:
Ping @bcipriano